app/serializers/alchemy/json_api/page_serializer.rb

```diff
@@ -24,17 +22,8 @@ class PageSerializer
 
       belongs_to :language, record_type: :language, serializer: ::Alchemy::JsonApi::LanguageSerializer
 
-      # All public elements of this page regardless of if they are fixed or nested.
-      # Used for eager loading and should be used as the +include+ parameter of your query
-      has_many :all_elements, record_type: :element, serializer: ELEMENT_SERIALIZER
-
-      # The top level public, non-fixed elements of this page that - if present -
-      # contains their nested_elements.
-      has_many :elements, record_type: :element, serializer: ELEMENT_SERIALIZER
-
-      # The top level public, fixed elements of this page that - if present -
-      # contains their nested_elements.
-      has_many :fixed_elements, record_type: :element, serializer: ELEMENT_SERIALIZER
+      # All visible elements of this page regardless of if they are fixed or nested.
+      has_many :elements, record_type: :element, serializer: ::Alchemy::JsonApi::ElementSerializer
```

Comment:
Let's keep the current API here rather than making changes to it. If `all_elements` is `included`, then `fixed_elements` and `elements` will be fully present as well.